### PR TITLE
alert utilization value is rounded and added units also

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bugid: Tendrl/monitoring-integration#226

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@r0h4n  @dahorak please review

File: tendrl/monitoring_integration/alert/handlers/cluster/gluster/brick_utilization_handler.py [hunk 1]
```diff
@@ -42,8 +42,8 @@ def format_alert(self, alert_json):
             if alert['severity'] == "WARNING":
                 alert['tags']['message'] = (
                     "Brick utilization of %s in node %s in "
```
Comment (shtripat) [inline tendrl/monitoring_integration/alert/handlers/cluster/gluster/brick_utilization_handler.py]: I feel its better to show the brick name as single entry in the form `{node-fqdn}:{brick path}`.
The message in current form looks too big.

**Comment by GowthamShanmugam** [comment]
@shtripat changes done

**Comment by r0h4n** [comment]
Verified?

**Comment by GowthamShanmugam** [comment]
@r0h4n It is verified

**Comment by GowthamShanmugam** [comment]
https://github.com/Tendrl/ui/issues/734

**Pull request merged**